api/rpc/account/account.go

```diff
@@ -27,6 +29,15 @@ func (s *Server) SignUp(ctx context.Context, in *SignUpRequest) (out *SessionRep
 	if err != nil {
 		return nil, grpc.Errorf(codes.Internal, "hashing error")
 	}
+	account := &schema.Account{
+		Id:         strconv.Itoa(len(accounts)),
+		Name:       in.Name,
+		Type:       schema.AccountType_USER,
+		Email:      in.Email,
+		PwHashcode: hash,
+		IsVerified: false,
+	}
+	accounts = append(accounts, account)
```

Comment:
You should be going through an interface that uses the mock data (and ultimately actual data) to give you `accounts` ... and that applies to all the usages of mock data. This would look like how we create an instance of the Storage interface with our etcd implementation. In this case, you would create an instance of the account access code using the mock data.